src/index.html

```diff
@@ -13,6 +13,52 @@
     />
   </head>
   <body>
-    <h1>Stars</h1>
+    <section class="stars--0 section">
```

Comment:
When we create a _Block_ with a _Modifier_ we need to pass both names to the class.
In that manner, the correct usage here would be:
```
<section class="stars stars--0">
```
There is also no need for the class "section", you can replace it with the _Block_ name "stars".
Please apply this change for all blocks in this file